src/Config.js

```diff
@@ -73,19 +73,31 @@ class Config {
   }
 
   static get groupsInSinopia() {
-    return [
-      ['ld4p', 'LD4P'],
-      ['pcc', 'PCC'],
-      ['cub', 'University of Colorado Boulder'],
-      ['cornell', 'Cornell University'],
-      ['harvard', 'Harvard University'],
-      ['nlm', 'National Library of Medicine'],
-      ['stanford', 'Stanford University'],
-      ['ucsd', 'University of California, San Diego'],
-      ['penn', 'University of Pennsylvania'],
-      ['hrc', 'Harry Ransom Center, University of Texas at Austin'],
-      ['wau', 'University of Washington'],
-    ]
+    return {
+      alberta: 'University of Alberta',
+      boulder: 'University of Colorado, Boulder',
+      chicago: 'University of Chicago',
+      cornell: 'Cornell University',
+      dlc: 'Library of Congress',
+      duke: 'Duke University',
+      frick: 'Frick Art Reference Library',
+      harvard: 'Harvard University',
+      hrc: 'University of Texas, Austin, Harry Ransom Center',
+      ld4p: 'LD4P',
+      michigan: 'University of Michigan',
+      minnesota: 'University of Minnesota',
+      nlm: 'National Library of Medicine',
+      northwestern: 'Northwestern University',
+      pcc: 'PCC',
+      penn: 'University of Pennsylvania',
+      princeton: 'Princeton University',
+      stanford: 'Stanford University',
+      tamu: 'Texas A&M University',
+      ucdavis: 'University of California, Davis',
+      ucsd: 'University of California, San Diego',
+      washington: 'University of Washington',
+      yale: 'Yale University',
+    }
```

Comment:
i just copied the structure seen here:
https://github.com/LD4P/sinopia_acl/blob/21284eb72cbe13012d700b225fec05b1b2a59bec/config/default.js#L24-L48

always unfortunate to have duplicated config, but this is at least a fix for the immediate issue that doesn't make things worse (now that the structure lines up, it's copying without reformatting, at least).

i'd considered getting the info from the server, since the `/repository` container lists all the groups (since the groups are all direct children).  unfortunately, that wouldn't contain the label info for each group, just the slug, and another request to trellis for _each_ group would be needed to get the labels that way (i think, unless there's a more clever way to query it that i'm not aware of).

two ideas i can think of off-hand to get rid of the config duplication:
* in sinopia_acl, put this config in a separate file, and pull a copy of that into this repo when building the editor image
* index group info to elastic search and ask elastic search for it

i think i prefer the latter of those two.

or, if we had a graph store (e.g. neo4j or neptune), we could do graphier queries.

but hopefully this'll do for now.

Reply:
I'm fine with the duplication for now; might I suggest adding a comment about the duplication in both places?

Reply:
@ndushay already have a PR up for the comment in sinopia_acl (LD4P/sinopia_acl#75), and added a comment to that effect in this PR too